### PR TITLE
CROWN-186 BE - Sell a resource page backend and database entity updates

### PR DESCRIPTION
**Pull request by fossterer**
- Updated models for SupplierResource and ReceiverSupplier with new fields as noted from CROWN-185
- Updated setters and getters accordingly

### CODE REVIEW

File: .jhipster/SupplierResource.json [hunk 1]
```diff
@@ -1,48 +1,88 @@
...
-    "jpaMetamodelFiltering": false,
-    "fluentMethods": true,
-    "readOnly": false,
-    "embedded": false,
-    "clientRootFolder": "",
-    "applications": "*"
+	"name": "SupplierResource",
+	"fields": [
+		{
+			"fieldName": "resourceType",
```
Comment (sharmmad) [inline .jhipster/SupplierResource.json:5]: This is not needed as we have the relationship entity already

File: .jhipster/SupplierResource.json [hunk 2]
```diff
@@ -1,48 +1,88 @@
...
+		{
+			"fieldName": "productAssets",
+			"fieldType": "String"
+		},
+		{
+			"fieldName": "proofOfLife",
+			"fieldType": "String"
+		},
+		{
+			"fieldName": "publicationPermission",
```
Comment (sharmmad) [inline .jhipster/SupplierResource.json:55]: This will be a boolean

Comment (fossterer) [reply .jhipster/SupplierResource.json:55]: I am not sure what this jhipster's JSON file's language is. "Boolean", "boolean" - I didn't see any documentation and hence left it a simple "String"

I'm going with "Boolean". Let's see how it behaves

Comment (sharmmad) [reply .jhipster/SupplierResource.json:55]: Yes, Boolean should work

Comment (fossterer) [reply .jhipster/SupplierResource.json:51]: Ready now @sharmmad

**Pull request merged**